_docs/reference/release-notes.md

```diff
@@ -8,4 +8,17 @@ layout: docs
 type: markdown
 ---
 
-TBD
+## Istio v0.1
+
+Istio v0.1 is the initial release of Istio. It works in a single Kubernetes cluster with a small number of services. Istio v0.1 supports the following features:
+- Installation of Istio into a Kubernetes namespace with a single command.
+- Injection of Envoy proxies into Kubernetes pods using `istioctl kube-inject`.
+- Automatic traffic capture for Kubernetes pods using iptables.
+- In-cluster load balancing for HTTP, gRPC, and TCP traffic.
+- Basic Kubernetes Ingress and Egress support.
```

Comment:
Egress is a non standard feature, so no need to qualify.
Ingress available features: regex paths, wildcard hosts, multiple resources/paths, single TLS cert secret, LB to network endpoints rather than service VIP.
Ingress missing features: multiple TLS secrets (no server side SNI), no URI rewriting or any other extra annotations used by ingress controllers.
Overall, ingress is mostly covered to Kubernetes spec except multi-host TLS/SNI.